tests/integration/query/simple/with_order_test.go

```diff
@@ -451,3 +451,380 @@ func TestQuerySimple_WithMultipleOrderFields_ReturnsError(t *testing.T) {
 		executeTestCase(t, test)
 	}
 }
+
+func TestQuerySimple_WithAliasOrder_ShouldOrderResults(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple query with basic alias order ASC",
+		Actions: []any{
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "John",
+					"Age": 21
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Bob",
+					"Age": 32
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Carlo",
+					"Age": 55
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Alice",
+					"Age": 19
+				}`,
+			},
+			testUtils.Request{
+				Request: `query {
+					Users(order: {_alias: {UserAge: ASC}}) {
+						Name
+						UserAge: Age
+					}
+				}`,
+				Results: map[string]any{
+					"Users": []map[string]any{
+						{
+							"Name":    "Alice",
+							"UserAge": int64(19),
+						},
+						{
+							"Name":    "John",
+							"UserAge": int64(21),
+						},
+						{
+							"Name":    "Bob",
+							"UserAge": int64(32),
+						},
+						{
+							"Name":    "Carlo",
+							"UserAge": int64(55),
+						},
+					},
+				},
+			},
+		},
+	}
+
+	executeTestCase(t, test)
+}
+
+func TestQuerySimple_WithAliasOrderOnNonAliasedField_ShouldOrderResults(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple query with basic alias order on non aliased field ASC",
+		Actions: []any{
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "John",
+					"Age": 21
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Bob",
+					"Age": 32
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Carlo",
+					"Age": 55
+				}`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"Name": "Alice",
+					"Age": 19
+				}`,
+			},
+			testUtils.Request{
+				Request: `query {
+					Users(order: {_alias: {Age: ASC}}) {
```

Comment:
thought: It would be nice to see a test with a compound order query that includes an alias.

Reply:
added a compound test